src/AppInstallerCLITests/TestData/MultiFileManifestV1_7/ManifestV1_7-MultiFile-Installer.yaml

```diff
@@ -19,6 +19,7 @@ InstallerSwitches:
   Log: /log=<LOGPATH>
   InstallLocation: /dir=<INSTALLPATH>
   Upgrade: /upgrade
+  Repair: /repair
```

Comment:
singleton manifest is still supported, please add corresponding entries totest singleton manifest as well